### PR TITLE
Add ability to install additional packages on GHA.

### PR DESCRIPTION
**Pull request by icemac**
This is needed for https://github.com/zopefoundation/z3c.layer.ready2go/pull/1
where lxml cloud not be installed on PyPy3 without liblxml and libxslt
dev packages installed.

### CODE REVIEW

File: config/README.rst [hunk 1]
```diff
@@ -198,6 +198,11 @@ updated. Example:
         "src/foo/bar.mo",
         ]
 
+    [github-actions]
+    additional-install = [
+        "sudo apt-get install libxml2-dev libxslt-dev"
```
Comment (sallner) [inline config/README.rst:203]:
```suggestion
        "sudo apt-get install libxml2-dev libxslt-dev",
```

**Review by sallner — APPROVED** [review]
The list is growing :)

File: config/README.rst (hunk 1 above)
Comment (mgedmin) [reply config/README.rst:203]: Shouldn't this be `sudo apt-get install -y ...`?

Also, I once had my GHA fail with 404 errors from sudo apt-get install due to outdated package lists and so had to prepend a `sudo apt-get update &&` to the command.

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**